docs/VaultNotaryContract_1.0.0.md

```diff
@@ -0,0 +1,196 @@
+### VaultNotary Contract 1.0.0
+
+The VaultNotary contract contains the functions to create, update and read the URI and Hash 
+of vaults, and the functions that manages the access control of addresses to the setter functions. 
+
+This is the first version of this contract, i.e. Version 1.0.0. 
+
+The version has the following endpoints,
+
+####1. Record Creation
+
+##### Register a Vault
+```JSON
+{
+	"method": "notary.register_vault_tx",
+	"params": {
+		"vaultId": "{{vault_id}}",
+		"senderWallet": "{{wallet_id}}", 
+		"vaultUri": "s3://my-bucket/2ed96ba9-26d4-4f26-b3da-c45562268480/meta.json", 
+		"vaultHash": "0x0f021e716c58d1e53222bf1cf9dfe8740470ef8d4a125499b26de58049688563"
+	},
+	"jsonrpc": "2.0",
+	"id": 0
+}
+```
+
+Note both vaultId and senderWallet are UUIDs. The response of this call also has a
+`contractVersion` field similar to the `createNewShipment` endpoint in the Load contract. 
+ 
+An example of a response, 
+```JSON
+{
+    "jsonrpc": "2.0",
+    "id": 0,
+    "result": {
+        "success": true,
+        "contractVersion": "1.0.0",
+        "transaction": {
+            "nonce": "0x00",
+            "chainId": 1337,
+            "data": "0x885bcd360000000000004000b00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000003d73333a2f2f6d792d6275636b65742f32656439366261392d323664342d346632362d623364612d6334353536323236383438302f6d6574612e6a736f6e0000000000000000000000000000000000000000000000000000000000000000000042307830663032316537313663353864316535333232326266316366396466653837343034373065663864346131323534393962323664653538303439363838353633000000000000000000000000000000000000000000000000000000000000",
+            "to": "0xa2207BB135287a4EB3ae4De32A0b99d112ae57B0",
+            "gasPrice": "0x0430e23400",
+            "gasLimit": "0x0637e0",
+            "value": "0x00"
+        }
+    }
+}
+```
+
+####2. URI and Hash Getter
+
+##### Get the Vault Information
+```JSON
+{
+	"method": "notary.1.0.0.get_vault_notary_details",
+	"params": {
+		"vaultId": "{{vault_id}}"
+	},
+	"jsonrpc": "2.0",
+	"id": 0
+}
+```
+
+An example response,
+```JSON
+{
+    "jsonrpc": "2.0",
+    "id": 0,
+    "result": {
+        "success": true,
+        "details": {
+            "0": "s3://my-bucket/2ed96ba9-26d4-4f26-b3da-c45562268480/meta.json",
+            "1": "0x0f021e716c58d1e53222bf1cf9dfe8740470ef8d4a125499b26de58049688563",
+            "vaultUri": "s3://my-bucket/2ed96ba9-26d4-4f26-b3da-c45562268480/meta.json",
+            "vaultHash": "0x0f021e716c58d1e53222bf1cf9dfe8740470ef8d4a125499b26de58049688563"
```

Comment:
Update the example with the new response

Reply:
fixed